packaging/aznfs/RPM/aznfs.spec

```diff
@@ -180,9 +183,10 @@ NORMAL="\e[0m"
 if [ $1 == 0 ]; then
 	# Verify if any existing mounts are there, warn the user about this.
 	existing_mounts=$(cat /opt/microsoft/aznfs/data/mountmap 2>/dev/null | egrep '^\S+' | wc -l)
```

Comment:
for better readability here, there should be some consistency with var names with existing_mounts, existing_mounts_for_nfsv4FilesShares. Try maybe existing_mounts_for_nfsv3 and similar like for v4. Others can suggest!

Reply:
same change in DEBIAN/prerm